models/zebra_motionworks.py

```diff
@@ -0,0 +1,9 @@
+from google.appengine.ext import ndb
+
+
+class ZebraMotionWorks(ndb.Model):
+    """
+    The ZebraMotionWorks model represents robot tracking data from the
+    Zebra MotionWorks system
+    """
```

Comment:
many want to include the schema in this comment as well?

Reply:
Schema is described in the parser. Maybe I'll make a note of it.